### PR TITLE
Add initial RHEL 6 preupg analysis.

### PR DESCRIPTION
**Pull request by jeffmcutter**

### CODE REVIEW

File: roles/analysis/tasks/main.yml [hunk 1]
```diff
@@ -19,12 +19,19 @@
     content: "{{ ansible_facts | ansible.builtin.combine({'ansible_local': {}}) }}"
     dest: /etc/ansible/facts.d/pre_ripu.fact
 
-# TODO: Include tasks for RHEL 6 to 7 non-leapp upgrade
-# - name: Include tasks for preupg assistant analysis
-#   ansible.builtin.include_tasks: analysis-preupg.yml
-#   when: ansible_distribution_major_version|int < 7
+- name: Include tasks for preupg assistant analysis
+  ansible.builtin.include_tasks: analysis-preupg.yml
```
Comment (djdanielsson) [inline roles/analysis/tasks/main.yml:23]: rename the file to have rhel6 if it is specifically for rhel 6

**Review by djdanielsson — CHANGES_REQUESTED** [review]

**Review by djdanielsson — APPROVED** [review]
LGTM

File: roles/analysis/handlers/main.yml [hunk 2]
```diff
@@ -6,4 +6,10 @@
     activationkey: "{{ satellite_activation_key_pre_leapp }}"
     org_id: "{{ satellite_organization }}"
     force_register: true
+
+- name: Remove lynx package
```
Comment (swapdisk) [inline roles/analysis/handlers/main.yml:10]: @jeffmcutter, why are we removing this package during analysis? Ideally, the analysis role shouldn't be making changes beyond installing the packages required to generate the report.

Comment (swapdisk) [reply roles/analysis/handlers/main.yml:10]: Got it, thanks!

**Pull request merged**